maintenance/rebuildLocalisationCache.php

```diff
@@ -151,6 +158,28 @@ function ( $item ) use ( $firstCodes ) {
 	}
 
 	/**
+	 * Helper function to include all localisation files to localisation cache rebuild process.
+	 * It looks for i18n and alias files from whole code directory and registers them in global array.
+	 */
+	private function includeAllMessagesFiles() {
```

Comment:
As far I've checked there's now space between `(` and `)` if there's no params in method.